findbugs-exclude.xml

```diff
@@ -0,0 +1,5 @@
+<FindBugsFilter>
+     <Match>
+       <Class name="~.+\.Messages" />
```

Comment:
Found it. This is the [method](https://github.com/findbugsproject/findbugs/blob/264ae7baf890d2b347d91805c90057062b5dcb1e/findbugs/src/java/edu/umd/cs/findbugs/detect/Naming.java#L620) that checks for a bad method name in the Findbugs project. A wrong method name meets all following conditions:

- Length is >= than 2
- First character is a letter
- First character is upper case
- Second character is a letter
- Second character is lower case
- **Name does not contain `_`**

`maven-localizer-plugin` transforms `.` to `_`. Normally properties in `Messages.properties` contains `.` so I guess that's why we have not run into this yet. Will change the property name in this PR.